x-pack/test/saved_object_api_integration/security_and_spaces/apis/resolve_import_errors.ts

```diff
@@ -20,30 +20,52 @@ const {
   SPACE_2: { spaceId: SPACE_2_ID },
 } = SPACES;
 const { fail400, fail409 } = testCaseFailures;
+const newId = (condition?: boolean) => (condition !== false ? { successParam: 'newId' } : {});
 
-const createTestCases = (overwrite: boolean, spaceId: string) => {
+const createTestCases = (overwrite: boolean, duplicate: boolean, spaceId: string) => {
```

Comment:
When revisiting this test suite (and the analogs in `security_only` and `spaces_only`), I decided to omit test cases that don't make sense when thinking about resolving import errors. In essence, I only have test cases for objects that could/would have resulted in import errors in the first place.